src/app/projects/components/contact-field-group/contact-field-group.component.ts

```diff
@@ -92,4 +92,13 @@ export class ContactFieldGroupComponent implements OnInit {
     const formGroup: FormGroup = this.formHelper.toFormGroup(metadata.itemMetadata);
     formArray.insert(count, formGroup);
   }
+
+  getControlValueForMetadataItem(formGroup: FormGroup, metadataKey: string): any {
+    // Allows dot notation e.g. getControlValueForMetadataItem(fg, 'project_role.ontology')
+    let value = formGroup.value;
+    metadataKey.split('.').forEach(key => {
+      value = value[key]
+    });
+    return value;
+  }
```

Comment:
It would be **ideal** to have a unit test for this but I tried writing one and it's difficult, so I think it's fine as it is.
If you can make a quick unit test then great, otherwise move on.

Reply:
You're right, done!